src/drivers/mct/shr/seq_comm_mct.F90

```diff
@@ -20,8 +20,10 @@ module seq_comm_mct
   use shr_sys_mod    , only : shr_sys_abort, shr_sys_flush
   use shr_mpi_mod    , only : shr_mpi_chkerr, shr_mpi_bcast, shr_mpi_max
   use shr_file_mod   , only : shr_file_getUnit, shr_file_freeUnit
+#ifdef TIMING
```

Comment:
Add a comment.  This is for unit testing.  TIMING is on by defaut when compiling cime normally.  Unit tests for Fortran don't need gptl AND have TIMING undefined so add this ifdef for unit tests.